Dockerfile.node-cache

```diff
@@ -12,18 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ARG_FROM
-# Use --no-install-recommends to not install nftables and work around https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=956655
-# Use iptables-legacy only until https://bugzilla.netfilter.org/show_bug.cgi?id=1422 is resolved
-# once fixed we will switch to k8s.gcr.io/debian-iptables-$(ARCH) to choose iptables-legacy or iptables-nft at run time
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    iproute2 \
-    iptables \
-    ebtables \
```

Comment:
will the new base image also contain ebtables?

Reply:
Yes, but it's the nf_tables version by default, and there is no magic auto-detection script
```
$ podman run -ti --rm k8s.gcr.io/build-image/debian-iptables:buster-v1.3.0 ebtables -V
ebtables 1.8.5 (nf_tables)
```
I will restore the `update-alternatives --set ebtables /usr/sbin/ebtables-legacy` to keep the current install working

I understand that this ebtables rule is an ugly hack used when you are in bridge mode, it would be way cleaner to inject a routing rule in the bridge CNI config `ip r add 169.254.25.10 via <nodeip>` (to have the route in each pod) and not hijack the traffic

Reply:
added, rebased, build tested

Reply:
```
$ docker run --rm -it --entrypoint=ebtables champtar/k8s-dns-node-cache-amd64:1.15.15-3-g8055b8f
ebtables v2.0.11 (legacy) (December 2011)
...
```

Reply:
> Yes, but it's the nf_tables version by default, and there is no magic auto-detection script
> 
> ```
> $ podman run -ti --rm k8s.gcr.io/build-image/debian-iptables:buster-v1.3.0 ebtables -V
> ebtables 1.8.5 (nf_tables)
> ```
> 
> I will restore the `update-alternatives --set ebtables /usr/sbin/ebtables-legacy` to keep the current install working
> 
> I understand that this ebtables rule is an ugly hack used when you are in bridge mode, it would be way cleaner to inject a routing rule in the bridge CNI config `ip r add 169.254.25.10 via <nodeip>` (to have the route in each pod) and not hijack the traffic

@champtar that would not work unless further changes were made (i.e. resolv.conf) with the downside of leaving all the applications broken in case only nodelocaldns is uninstalled without reverting the other changes that are not controlled by nodelocaldns.